app/components/UserOp/SvmRawTransaction.tsx

```
@@ -0,0 +1,539 @@
+                    {isRefreshing ? (
+                      <span>Refreshing...</span>
+                    ) : (
+                      <>
+                        <RefreshIcon /> Refresh Status
+                      </>
+                    )}
+                  </button>
+                </div>
+              )}
+            </>
+          )}
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)